tools/rnc-validate-format.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import subprocess
+import argparse
+from shutil import which
+from glob import glob
+
+# to validate and pretty print the rnc schema files before commit:
+# tools/rnc-validate-format.py
+
+parser = argparse.ArgumentParser(description="process csl schema files")
+parser.add_argument("-c", action="store_true", help="commit the file(s)")
+args = parser.parse_args()
+
+
+def rnc_format(rncfile, check=False):
+    try:
+        which("trang") is not None
+    except Exception:
+        print("trang commnd not found; please install")
+
+    rng_new = rncfile + "_new.rng"
+    rnc_new = rncfile + "_new.rnc"
+
+    # round-trip the rnc file through trang
+    subprocess.run(["trang", rncfile, rng_new])
+
+    if check:
```

Comment:
not sure this is the right approach; it's assuming moving back to the pre-commit mere;y checking the formatting, and I'm not even sure this is the best way to do that. should probably remove the `if`.